openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -664,7 +663,7 @@ def test_satisfy_all_requirements(self):
         self.assertFalse(api.is_user_eligible_for_credit(user.username, self.course_key))
 
         # Satisfy the other requirement
-        with self.assertNumQueries(24):
+        with self.assertNumQueries(23):
```

Comment:
Any idea why this changed?

Reply:
I think it has to do with moving the creation of `self.course` to `setUp` but I honestly couldn't tell you precisely why.